### PR TITLE
add CrvStep

### PR DESCRIPTION
**Pull request by rlcee**
Add CrvStep to Print and Validation

### CODE REVIEW

**Review by brownd1978 — APPROVED** [review]
looks good

File: Validation/src/ValCrvStep.cc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  _hN->Fill(coll.size()); 
+  for(auto cs : coll) {
+    _hb->Fill(cs.barIndex().asInt());
+    _ht->Fill(cs.startTime());
+    _hE->Fill(cs.visibleEDep());
+    _hposx->Fill(cs.startPosition().x());
+    _hposy->Fill(cs.startPosition().y());
+    _hposz->Fill(cs.startPosition().z());
+    _hp->Fill(cs.startMomentum().mag());
+    _hp2->Fill(cs.startMomentum().mag());
```
Comment (ehrlich-uva) [inline Validation/src/ValCrvStep.cc:36]: _hp and _hp2 seems to be filled with the same number.

File: Print/fcl/printLong.fcl [hunk 2]
```diff
@@ -72,12 +72,17 @@ physics :{
 	#inputTags : [ "CaloClusterMaker" ] # select instances
       }
 
+      crvStepPrinter : {
+        verbose: 1      # 0.1
+        eCut : -10.0          # cut on hits
```
Comment (ehrlich-uva) [inline Print/fcl/printLong.fcl:77]: Is this negative sign correct (no energy cut)?

File: Validation/src/ValCrvStep.cc (hunk 1 above)
Comment (rlcee) [reply Validation/src/ValCrvStep.cc:36]: Yes, the two plots have very different scales - beam scale and cosmics scale.

File: Print/fcl/printLong.fcl (hunk 2 above)
Comment (rlcee) [reply Print/fcl/printLong.fcl:77]: Yes, negative means effectively no cut.  It is here as a reminder that the cut is possible.

File: Validation/src/ValCrvStep.cc (hunk 1 above)
Comment (ehrlich-uva) [reply Validation/src/ValCrvStep.cc:36]: Ok. Thank you.

File: Print/fcl/printLong.fcl (hunk 2 above)
Comment (ehrlich-uva) [reply Print/fcl/printLong.fcl:77]: Ok

**Review by ehrlich-uva — APPROVED** [review]

**Pull request merged**